bin/i18n/i18n_script_utils.rb

```diff
@@ -174,33 +179,28 @@ def self.get_script_level(route_params, url)
   # urls; we expect to expand this function over time as new cases are
   # discovered.
   def self.get_level_from_url(url)
-    # memoize to reduce repeated database interactions
-    @levels_by_url ||= Hash.new do |hash, new_url|
```

Comment:
can you explain a bit about this change?

Reply:
Reverted, sorry